addons/web/static/src/js/widgets/notification.js

```diff
@@ -51,6 +51,8 @@ var Notification = Widget.extend({
         this.type = params.type === undefined ? 'warning' : params.type;
         this.className = params.className || '';
         this._closeCallback = params.onClose;
+        this.headerIconDisable = !!params.headerIconDisable;
```

Comment:
suggested name: `hasHeaderIcon`

reasons:
- makes it more clear it is a boolean (starts with "is" or "has")
- the meaning of "disabling" the icon is not clear
- use "positive" option instead of "negative"

Reply:
As this name suggestion, now could we put like "this.hasHeaderIcon = params.hasHeaderIcon === undefined ? true : params.hasHeaderIcon"...?

Reply:
I think it's cleaner with de-structuring with default values.
```js
const { headerIconDisable = true } = param;
```
Also don't forget to add default value in docstring.